assembl/static2/css/components/header.scss

```diff
@@ -272,14 +272,19 @@
   }
 }
 
-@media screen and (max-width: 375px) {
+@media screen and (max-width: $screen-sm-min) {
   .header-section {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+
     .statistic {
       .intermediary-container {
         padding: 20px 0;
 
         .stat-container {
           margin-top: 5px;
+          padding: 0 9px !important;
```

Comment:
ay, no me gusta `!important`. Are there top level padding that you are overriding?